prepublish

```diff
@@ -1,28 +1,32 @@
 #!/bin/bash
 
-rm -rvf *-10m.json
+PATH=$PATH:./node_modules/.bin:
```

Comment:
Load `$PATH` variables from node bin. This ensures the scripts works on any machine after `yarn install` is run